app/Template/customfilter/index.php

```diff
@@ -0,0 +1,38 @@
+<div class="page-header">
+    <h2><?= t('Custom filters') ?></h2>
+</div>
+<div>
+    <table>
+        <tr>
+            <th><?= t('Filter') ?></th>
+            <th><?= t('Name') ?></th>
+            <th><?= t('Shared') ?></th>
+            <th><?= t('Created by') ?></th>
+            <th><?= t('Actions') ?></th>
+        </tr>
```

Comment:
Do not display the table header if there is no custom filters, the page layout is weird otherwise